README.md

```diff
@@ -15,14 +17,20 @@ Detailed documentation can be found in our [GitHub Documenation Page](http://vmw
 
 ## Download
 
-**[Click here to download (Github releases)] (https://github.com/vmware/docker-volume-vsphere/releases)**
+**Click here to download [Github releases](https://github.com/vmware/docker-volume-vsphere/releases)**
```

Comment:
original => ``` [Click here to download (Github releases)] (https://github.com/vmware/docker-volume-vsphere/releases)```
with this change => ```Click here to download Github releases```

Just a formatting change, please let me what you think.